app/classifier/tasks/survey/index.cjsx

```diff
@@ -75,15 +78,16 @@ module.exports = React.createClass
     @setState filters: {}
 
   clearSelection: ->
-    @setState 
+    @setState
       selectedChoiceID: ''
       focusedChoice: @state.selectedChoiceID
     newAnnotation = Object.assign {}, @props.annotation, _choiceInProgress: false
     @props.onChange newAnnotation
 
   handleAnnotation: (choice, answers, e) ->
```

Comment:
`choice` here is the selected choice ID, so you don't need `selectedChoiceID` later.